src/backing/client/grpc_client/proto_util.cc

```diff
@@ -74,17 +73,17 @@ StatusCode MapStatusCode(grpc::StatusCode const& code) {
 }  // namespace
 
 google::cloud::kms::v1::Digest MakeDigest(DigestCase type,
-                                          absl::string_view digest_bytes) {
+                                          std::string digest_bytes) {
   google::cloud::kms::v1::Digest proto_digest;
   switch (type) {
     case DigestCase::kSha256:
-      proto_digest.set_sha256(digest_bytes.data());
+      proto_digest.set_sha256(digest_bytes.data(), digest_bytes.length());
```

Comment:
You should just be able to write: `proto_digest.set_sha256(digest_bytes)` as long as `digest_bytes` is a `std::string` rather than an `absl::string_view`.

https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#proto3_string